<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(122, 65, 78);</string>
  </property>
  <widget class="QLabel" name="bw_image">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>481</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>290</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(163, 163, 163);</string>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>350</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(163, 163, 163);</string>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="bw_check">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>50</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Чёрно-белое</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>260</y>
     <width>121</width>
     <height>3</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(185, 93, 107);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
